From 1958c57db63bb37c4aaf9fc799bbd00b8179c8cf Mon Sep 17 00:00:00 2001
From: Doug Goldstein <doug.goldstein@rackspace.com>
Date: Thu, 1 Feb 2018 11:32:58 -0600
Subject: [PATCH 01/18] xenserver: fix Python errors in Citrix changes

The Citrix changes contribute some white space errors and syntax errors
which makes Python upset.
---
 ...sr_share_openvswitch_scripts_ovs-xapi-sync | 19 +++++++++++--------
 1 file changed, 11 insertions(+), 8 deletions(-)

diff --git a/xenserver/usr_share_openvswitch_scripts_ovs-xapi-sync b/xenserver/usr_share_openvswitch_scripts_ovs-xapi-sync
index a0aec7b9f..bf6cdf849 100755
--- a/xenserver/usr_share_openvswitch_scripts_ovs-xapi-sync
+++ b/xenserver/usr_share_openvswitch_scripts_ovs-xapi-sync
@@ -80,15 +80,17 @@ def init_session():
 def get_network_by_bridge(br_name):
     if not init_session():
         vlog.warn("Failed to get bridge id %s because"
-                " XAPI session could not be initialized" % br_name)
+                  " XAPI session could not be initialized" % br_name)
         return None
 
-    recs = session.xenapi.network.get_all_records_where('field "bridge"="%s"' % br_name)
+    qry = 'field "bridge"="%s"' % br_name
+    recs = session.xenapi.network.get_all_records_where(qry)
     if len(recs) > 0:
         return recs.values()[0]
 
     return None
 
+
 # There are possibilities when multiple xs-network-uuids are set for a bridge.
 # In cases like that, we should choose the bridge-id associated with the bridge
 # name.
@@ -103,6 +105,7 @@ def get_single_bridge_id(bridge_ids, br_name, default=None):
     xapi_down = True
     return default
 
+
 # By default, the "bridge-id" external id in the Bridge table is the
 # same as "xs-network-uuids".  This may be overridden by defining a
 # "nicira-bridge-id" key in the "other_config" field of the network
@@ -127,7 +130,7 @@ def get_iface_id(if_name, xs_vif_uuid):
 
     if not init_session():
         vlog.warn("Failed to get interface id %s because"
-                " XAPI session could not be initialized" % if_name)
+                  " XAPI session could not be initialized" % if_name)
         return xs_vif_uuid
 
     try:
@@ -151,7 +154,7 @@ def get_vm_id(if_name, xs_vm_uuid):
 
     if not init_session():
         vlog.warn("Failed to get vm id for interface id %s because"
-                " XAPI session could not be initialized" % if_name)
+                  " XAPI session could not be initialized" % if_name)
         return xs_vm_uuid
 
     try:
@@ -217,7 +220,7 @@ def update_in_band_mgmt(row):
     other_config = row.other_config
     if dib and dib not in ['true', 'false']:
         vlog.warn('"%s" isn\'t a valid setting for '
-                "other_config:disable-in-band on %s" % (dib, row.name))
+                  "other_config:disable-in-band on %s" % (dib, row.name))
     elif set_or_delete(other_config, 'disable-in-band', dib):
         row.other_config = other_config
 
@@ -227,7 +230,7 @@ def main():
 
     parser = argparse.ArgumentParser()
     parser.add_argument("database", metavar="DATABASE",
-            help="A socket on which ovsdb-server is listening.")
+                        help="A socket on which ovsdb-server is listening.")
     parser.add_argument("--root-prefix", metavar="DIR", default='',
                         help="Use DIR as alternate root directory"
                         " (for testing).")
@@ -267,7 +270,7 @@ def main():
     while True:
         unixctl_server.run()
         if exiting:
-            break;
+            break
 
         idl.run()
         if not xapi_down and not flush_cache and seqno == idl.change_seqno:
-- 
2.17.2 (Apple Git-113)
